shopinvader_product_stock/models/shopinvader_variant.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com)
+# Copyright 2018 ACSONE SA/NV
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+from collections import defaultdict
+
+
+class ShopinvaderVariant(models.Model):
+    _inherit = 'shopinvader.variant'
+
+    stock_data = fields.Serialized(compute='_compute_stock_data')
+
+    def _get_stock_key(self):
```

Comment:
This method introduce some confusion... Indeed into the next method you declare a field `stock_key` without relation with this method.... At first read I don't understand the meaning/purpose of this method.